src/docfx/build/Build.cs

```diff
@@ -24,23 +24,23 @@ internal static class Build
             var docset = new Docset(context, docsetPath, config, options);
 
             var tocMap = await BuildTableOfContents.BuildTocMap(docset.BuildScope);
-            var repo = new GitRepoInfoProvider(docset);
+            var contribution = ContributionInfo.Load(docset);
```

Comment:
either is fine with me. I choose load because it is more IO bound

---
In reply to: [199392427](https://github.com/dotnet/docfx/pull/2981#discussion_r199392427) [](ancestors = 199392427)